sdk/network/mgmt/src/main/java/com/azure/management/network/implementation/NetworkPeeringImpl.java

```diff
@@ -24,8 +24,7 @@
         NetworkPeering, Network, VirtualNetworkPeeringInner, NetworkPeeringImpl, NetworkManager>
     implements NetworkPeering,
         NetworkPeering.Definition,
-        NetworkPeering.Update,
-        IndependentChild.DefinitionStages.WithParentResource<NetworkPeering, Network> {
```

Comment:
Any effect on this deleted interface?

Reply:
No, the deletion is just because the super class has such implements